src/Microsoft.Win32.Primitives/tests/Win32Exception.cs

```diff
@@ -80,7 +80,7 @@ public static void InstantiateException()
         private const int E_FAIL = unchecked((int)0x80004005);
 
         [Fact]
-        [PlatformSpecific(PlatformID.Windows)]
+        [PlatformSpecific(Xunit.PlatformID.Windows)]
```

Comment:
`Xunit.PlatformID` is a `Flags` enum, so it really should be plural if following the framework design guidelines. Maybe `Platforms` or `PlatformIDs`?